[V3/develop] cog uninstall throws exception

# Command bugs

#### Command name

`cog uninstall`

#### What cog is this command from?

`Downloader`

#### What were you expecting to happen?

Remove cog from installed list

#### What actually happened?

When cog is unloaded, exception is raised:
```py
[2019-04-25 14:00:49] [ERROR] red: Exception in command 'cog uninstall'
Traceback (most recent call last):
  File "/home/fixator/Red-V3/lib/python3.7/site-packages/discord/ext/commands/core.py", line 63, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/fixator/Red-V3/lib/python3.7/site-packages/redbot/cogs/downloader/downloader.py", line 347, in _cog_uninstall
    ctx.bot.unload_extension(real_name)
  File "/home/fixator/Red-V3/lib/python3.7/site-packages/discord/ext/commands/bot.py", line 650, in unload_extension
    raise errors.ExtensionNotLoaded(name)
discord.ext.commands.errors.ExtensionNotLoaded: Extension 'rolemanagement' has not been loaded.
```

#### How can we reproduce this issue?

1. Unload custom cog
2. Try to uninstall that cog via `[p]cog uninstall`

--- a/redbot/cogs/downloader/downloader.py
+++ b/redbot/cogs/downloader/downloader.py
@@ -344,7 +344,8 @@ class Downloader(commands.Cog):
 
                 poss_installed_path = (await self.cog_install_path()) / real_name
                 if poss_installed_path.exists():
-                    ctx.bot.unload_extension(real_name)
+                    with contextlib.suppress(commands.ExtensionNotLoaded):
+                        ctx.bot.unload_extension(real_name)
                     await self._delete_cog(poss_installed_path)
                     uninstalled_cogs.append(inline(real_name))
                 else:

